pdfminer/pdffont.py

```diff
@@ -76,7 +78,33 @@ def get_widths2(seq):
 #assert get_widths2([1,2,3,4,5]) == {1:(3, (4,5)), 2:(3, (4,5))}
 #assert get_widths2([1,[2,3,4,5],6,[7,8,9]]) == {1:(2, (3,4)), 6:(7, (8,9))}
 
-
+def computeStat(it):
+	stat=defaultdict(lambda: defaultdict(int))
+	for fd in it:
+		for cat, size in itemz(fd):
+			stat[cat][size]+=1
+	metrics={}
+	for cat, sizes in stat.items():
+		res=max(sizes.items(), key=lambda x: x[1])
+		if res[1] == 1:
+			metrics[cat]=mean(sizes.keys())
+		else:
+			metrics[cat]=res[0]
+	return metrics
+
+def computeFallbackFontMetrics(FONT_METRICS):
```

Comment:
This function is rather complicated. Could you add a docstring that describe what it does?

Reply:
Unfortunately I don't remember why I have implemented this this way. So I can only to try to RE it from the code.